src/fides/api/ops/service/connectors/fides/fides_client.py

```diff
@@ -0,0 +1,173 @@
+    PrivacyRequestStatus.error,
+    PrivacyRequestStatus.denied,
+]
+
+
+class FidesClient:
+    """
+    A helper client class to broker communications between Fides servers.
+    """
+
+    def __init__(
+        self,
+        uri: str,
+        username: str,
+        password: str,
+    ):
+        self.session = Session()
+        self.uri = uri
+        self.username = username
+        self.password = password
+        self.token = None
+
+    def login(self) -> None:
+        ul: UserLogin = UserLogin(username=self.username, password=self.password)
+        response = requests.post(
+            self.uri + urls.V1_URL_PREFIX + urls.LOGIN, json=ul.dict()
+        )
+        if response.ok:
+            self.token = response.json()["token_data"]["access_token"]
+        else:
+            log.error(f"Error logging in on remote Fides {self.uri}")
+            response.raise_for_status()
+
+    def authenticated_request(
+        self,
+        method: str,
+        path: str,
+        headers: Optional[Dict[str, Any]] = {},
+        query_params: Optional[Dict[str, Any]] = {},
+        data: Optional[Any] = None,
+        json: Optional[Any] = None,
+    ) -> PreparedRequest:
+
+        if not self.token:
+            raise FidesError(
+                f"Unable to create authenticated request. No token for Fides connector for server {self.uri}"
+            )
+
+        req: PreparedRequest = Request(
+            method=method,
+            url=f"{self.uri}{path}",
+            headers=headers,
+            params=query_params,
+            data=data,
+            json=json,
+        ).prepare()
+        req.headers["Authorization"] = "Bearer " + self.token
+        return req
+
+    def create_privacy_request(
+        self, external_id: Optional[str], identity: Identity, policy_key: str
+    ) -> str:
+        """
+        Create privacy request on remote fides by hitting privacy request endpoint
+        Retruns the created privacy request ID
+        """
+        pr: PrivacyRequestCreate = PrivacyRequestCreate(
+            external_id=external_id,
+            identity=identity,
+            policy_key=policy_key,
+        )
+
+        request: PreparedRequest = self.authenticated_request(
+            method="POST",
+            path=urls.V1_URL_PREFIX + urls.PRIVACY_REQUEST_AUTHENTICATED,
+            json=[pr.dict()],
+        )
+        response = self.session.send(request)
+        if not response.ok:
+            log.error(f"Error creating privacy request on remote Fides {self.uri}")
+            response.raise_for_status()
+        if response.json()["failed"]:
+            # TODO better handle errored state here?
+            raise FidesError(
+                f"Failed privacy request creation on remote Fides {self.uri} with failure message: {response.json()['failed'][0]['message']}"
+            )
+        return response.json()["succeeded"][0]["id"]
+
+    def poll_for_request_completion(
```

Comment:
need to figure out how to reconcile with https://github.com/ethyca/fides/pull/1860 - shouldn't be hard, but there are pieces from both that i think we should incorporate